servers/rendering/rasterizer_rd/rasterizer_scene_rd.cpp

```diff
@@ -181,10 +181,9 @@ void RasterizerSceneRD::_update_reflection_mipmaps(ReflectionData &rd) {
 			for (int j = 0; j < rd.layers[i].mipmaps.size() - 1; j++) {
 				for (int k = 0; k < 6; k++) {
```

Comment:
Couldn't we optimize this by rendering the mipmaps for all 6 faces at once? ReflectionData now stores a per-miplevel cubemap view that we can use to write to an entire mipmap level at once https://github.com/godotengine/godot/blob/636d4bad99db3293cb3b7ac13b7f7e3c69a83296/servers/rendering/rasterizer_rd/rasterizer_scene_rd.h#L120

Reply:
technically you can run compute shaders in parallel, then synchronize with barrier, but not quite sure what you have in mind... might be useful in a separate PR i guess, in this one I only converted stuff, but did not optimize anything in particular.

Reply:
Sure, I can do later. Instead of running make_mipmap once for each face, we call it it once per mip level, and then adjust the compute shader to run for a full cubemap instead of per face. That way we can invoke the shader 1/6th as many times. 

Its an optimization that can take place later for sure though.